docs/tutorial/piezotracking.rst

```diff
@@ -0,0 +1,164 @@
+.. warning::
+    This is alpha functionality. While usable, this has not yet been tested in a large
+    number of different scenarios. The API may also still be subject to change.
+
+Piezo tracking
+==============
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+In this tutorial, we will determine the high frequency distance (piezo distance) between the beads from the piezo mirror position of trap 1 and the corresponding force data.
+We will also show how to use the same reference curve to apply baseline correction for force signals in post processing.
+
+Trap Positional Calibration
+---------------------------
+
+The first step is to calibrate the high frequency trap position from the piezo mirror to the low frequency bead-to-bead distance measured by the camera.
+
+For this calibration, we require a dataset (acquired in the absence of a tether) in which trap 1 is moved over the entire distance range intended for the experiment.
+The exported data file must contain the trap 1 position and camera-based distance channels.
+
+*Note that this dataset should be acquired such that the bead tracking templates do not overlap. In case of overlap, make sure to first slice the position and distance data such that this region is not included.*
+Let's load this dataset and perform the distance calibration by invoking::
```

Comment:
```suggestion
*Note that this dataset should be acquired such that the bead tracking templates do not overlap. In case of overlap, make sure to first slice the position and distance data such that this region is not included.*

Let's load this dataset and perform the distance calibration by invoking::
```

Reply:
👍